packages/dds/tree/.eslintrc.cjs

```diff
@@ -39,4 +39,33 @@ module.exports = {
 			},
 		},
 	],
+	settings: {
```

Comment:
changes to this eslint and sequence's eslint will be removable once the changes to the base config are properly staged. I'm including them here as well to see what other tooling this approach breaks. (Hopefully none, but there are always surprises with this kind of thing)